velox/functions/prestosql/tests/InPredicateTest.cpp

```diff
@@ -189,6 +189,20 @@ TEST_F(InPredicateTest, tinyint) {
   testsIntegerConstant<int8_t>();
 }
 
+TEST_F(InPredicateTest, boolean) {
+  auto input = makeRowVector({
+      makeFlatVector<bool>({true, false}, BOOLEAN()),
+  });
+
+  std::string predicate = "c0 IN (TRUE, FALSE)";
```

Comment:
This is confusing to a reader because they may think why would anyone want to test whether a boolean is true or false. Perhaps, add a comment and add input null value.